packages/components/src/components/primitives/CircularProgress/types.ts

```diff
@@ -0,0 +1,6 @@
+import { CircularProgressProps as ChakraCircularProgressProps } from '@chakra-ui/react';
+
+
```

Comment:
can you confirm do you have the prettier extension installed and active?, I've been seeing styling issues in your PRs that should be automatically fixed by prettier

Reply:
i have fixed the formatting errors.
i have the extension, but the _format on save_ option was disabled!